charts/icm-as/templates/as-deployment.yaml

```diff
@@ -61,10 +61,18 @@ spec:
             value: "false"
           - name: SERVER_NAME
             value: "appserver"
-          {{- if .Values.debug.enabled }}
+          {{- if .Values.jvm.debug.enabled }}
```

Comment:
that modification looks like an API break for existing charts. so after release 1.0.0 this would be a major change.

Reply:
Yes, of cause. But until then we can make those changes without hesitating (I just moved the debug.enabled flag into the jvm context because it belongs to the jvm).